gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -191,7 +193,20 @@ private void showAddFeatureDialog(Pair<ImmutableList<Layer>, Point> args) {
               viewModel.addFeature(layer, point);
               break;
             case POLYGON:
-              showPolygonInfoDialog(layer);
+              if (localValueStore.isPolygonDialogInfoShown()) {
+                viewModel
+                    .getActiveProject()
+                    .ifPresentOrElse(
+                        project -> {
+                          polygonDrawingViewModel.startDrawingFlow(project, layer);
+                          mapContainerViewModel.setViewMode(Mode.DRAW_POLYGON);
+                        },
+                        () -> {
```

Comment:
Let's log an error that there's no active project (not "empty project") and continue instead of crashing the app.

Reply:
updated.